### PR TITLE
Debug jinja render, hiddenjinjatoggle, and retry functionality

### PR DESCRIPTION
**Pull request by sHermanGriffiths**
# Author's Checklist

Before assigning reviewers, step through the following checklist. Check each item once it's completed. If an item is skipped, write an explanation below the item.

- [x] **Construction:** Review the code diff and confirm there's no dead code, debug code, or unnecessary comments.
- [x] **Requirements:** Review the requirements in the issue for this PR and confirm they're all implemented.
- [x] **Architecture:** Confirm that the changes to the code are consistent with the existing architecture.
- [ ] **SOUP/OTS Software:** Any new dependencies have appropriate licenses and are documented or pinned in requirements files.
  N/A
- [ ] **Unit Test:** Atomic unit tests have been added, if appropriate.
  N/A
- [ ] **End-to-End Tests:** End-to-end tests have been extended or updated, if appropriate.
  N/A
- [ ] **Low-level Documentation:** Comments or doc strings for affected code have been updated.
  N/A
- [ ] **High-Level Documentation:** The README is updated, as appropriate.
  N/A
- [ ] **Change Log:** New features or bug fixes have been added to the change log in the README.
  N/A

### CODE REVIEW

File: n2y/plugins/hiddenjinjatoggles.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 class HiddenJinjaToggleBlock(ToggleBlock):
```
Comment (johndgiese) [inline n2y/plugins/hiddenjinjatoggles.py:12]: Can you rename this to "ExpandBlueToggleBlock". Also can you rename the file to "expandbluetoggles.py"

File: n2y/plugins/expandbluetoggles.py [hunk 2]
```diff
@@ -9,16 +9,16 @@
 logger = logging.getLogger(__name__)
 
 
-class HiddenJinjaToggleBlock(ToggleBlock):
+class ExpandBlueToggleBlock(ToggleBlock):
     def __init__(self, client, notion_data, page, get_children=True):
         super().__init__(client, notion_data, page, get_children)
-        if self.notion_type_data["color"] != "blue":
+        if self.notion_type_data["color"] != "blue_background":
```
Comment (bimbashrestha) [inline n2y/plugins/expandbluetoggles.py:15]: We'll need to update the test for this as well: [tests/test_hiddenjinjatoggles.py](https://github.com/innolitics/n2y/pull/102/files#diff-6fd060877e313b38e5a31293414cd3517041c4b9bdb406eefe42c3e20501ddb5)

File: n2y/plugins/jinjarenderpage.py [hunk 3]
```diff
@@ -158,16 +158,24 @@ def to_pandoc(self):
 
     def _render(self, ast):
         context = self.client.plugin_data['render_context']
-
         config = {'md_extensions': ['jinja2.ext.do']}
+        parent_id = self.notion_parent['database_id'].replace('-', '')
```
Comment (bimbashrestha) [inline n2y/plugins/jinjarenderpage.py:162]: Use https://github.com/innolitics/n2y/blob/main/n2y/utils.py#L136-L138 please

File: n2y/notion.py [hunk 4]
```diff
@@ -61,15 +61,17 @@ def wrapper(*args, **kwargs):
             client.retry_count = 0
             return response
         except APIResponseError as err:
-            can_retry = 'retry-after' in err.headers
-            if can_retry and client.retry_api_calls:
+            rate_limited = err.code == 'rate_limited'
+            retry_after = 'retry-after' in err.headers
```
Comment (bimbashrestha) [inline n2y/notion.py:65]: This variable naming is a bit confusing to me. `retry_after` is a boolean but we also later use it as the amount to sleep by. Maybe we can just use two variables here: one as the boolean to see if we have `retry-after` in the headers and one that stores the amount to sleep by?

**Review by bimbashrestha — CHANGES_REQUESTED** [review]

**Comment by bimbashrestha** [comment]
Changes look good! Just get CI passing and this is good to go

**Review by bimbashrestha — APPROVED** [review]
Nice work!

**Pull request merged**